packages/flp-config-inquirer/src/prompts/utils.ts

```diff
@@ -0,0 +1,34 @@
+import { isAppStudio } from '@sap-ux/btp-utils';
+
+interface Platform {
+    name: string;
+    technical: string;
+}
+
+export const PLATFORMS: { [key: string]: Platform } = {
+    VSCODE: {
+        name: 'Visual Studio Code',
+        technical: 'VSCode'
+    },
+    SBAS: {
+        name: 'SAP Business Application Studio',
+        technical: 'SBAS'
+    },
+    CLI: {
+        name: 'CLI',
+        technical: 'CLI'
+    }
+};
+
+/**
+ * Determines the platform where the application is running.
+ *
+ * @returns {Platform} An object containing the platform's name and technical identifier.
+ */
+export function getPlatform(): Platform {
+    if (process.mainModule?.filename.includes('yo') || process.stdin.isTTY) {
+        return PLATFORMS.CLI;
+    } else {
+        return isAppStudio() ? PLATFORMS.SBAS : PLATFORMS.VSCODE;
+    }
+}
```

Comment:
Maybe this method should be extracted to `environment-check` package?

Reply:
Please check `getHostEnvironment` in `@sap-ux/fiori-generator-shared`

Reply:
Changed in [1080bf8](https://github.com/SAP/open-ux-tools/pull/2580/commits/1080bf8852c6eedf94422dc2d0479d2642c704e6)